bears/general/TextLintBear.py

```diff
@@ -0,0 +1,125 @@
+import json
+
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+from dependency_management.requirements.PipRequirement import PipRequirement
+
+
+@linter(executable='textlint',
+        output_format='regex',
+        output_regex=r'(?P<line>\d+):(?P<column>\d+)[\s*|✓]*(?P<severity>'
+                     r'error|warning)\s*(?P<message>.+)')
+class TextLintBear:
+    """
+    The pluggable linting tool for text and markdown. It is similar to
+    ESLint, but textlint for natural language.
+    """
+
+    LANGUAGES = {'HTML', 'Markdown', 'reStructuredText'}
+    REQUIREMENTS = {NpmRequirement('textlint', '7.3.0'),
+                    NpmRequirement('textlint-plugin-html', '0.1.5'),
+                    NpmRequirement('textlint-plugin-rst', '0.1.1'),
+                    NpmRequirement('textlint-rule-alex', '1.2.0'),
+                    NpmRequirement('textlint-rule-common-misspellings',
+                                   '1.0.1'),
+                    NpmRequirement('textlint-rule-date-weekday-mismatch',
+                                   '1.0.5'),
+                    NpmRequirement('textlint-rule-ginger', '2.1.0'),
+                    NpmRequirement('textlint-rule-max-comma', '1.0.4'),
+                    NpmRequirement('textlint-rule-max-number-of-lines',
+                                   '1.0.3'),
+                    NpmRequirement('textlint-rule-ng-word', '1.0.0'),
+                    NpmRequirement('textlint-rule-no-dead-link', '3.1.1'),
+                    NpmRequirement('textlint-rule-no-empty-section', '1.1.0'),
+                    NpmRequirement('textlint-rule-no-nfd', '1.0.1'),
+                    NpmRequirement('textlint-rule-no-start-'
+                                   'duplicated-conjunction', '1.1.3'),
+                    NpmRequirement('textlint-rule-no-surrogate-pair',
+                                   '1.0.1'),
+                    NpmRequirement('textlint-rule-no-todo', '2.0.0'),
+                    NpmRequirement('textlint-rule-period-in-list-item',
+                                   '0.2.0'),
+                    NpmRequirement('textlint-rule-rousseau', '1.4.5'),
+                    NpmRequirement('textlint-rule-unexpanded-acronym',
+                                   '1.2.1'),
+                    NpmRequirement('textlint-rule-write-good', '1.6.0'),
+                    PipRequirement('docutils-ast-writer', '0.1.2')}
```

Comment:
Dependency for the rst plugin, https://github.com/jimo1001/textlint-plugin-rst#installation